package/tests/test_models/test_graph/test_graph_nodes.py

```diff
@@ -384,9 +384,8 @@ def test_partitioned_data_node_metadata(self):
         data_node_metadata = DataNodeMetadata(data_node=data_node)
         assert data_node_metadata.filepath == "partitioned/"
 
-    @patch("builtins.__import__", side_effect=import_mock)
```

Comment:
Why was this removed? Seems to me that we should be able to run the tests without plotly installed (related to the point of removing plotly from requirements.txt).

Reply:
I'm not sure. @rashidakanchwala? Should I add it back in?

Reply:
so I removed it, because it wasn't actually being used. :S